format/spec.md

```diff
@@ -951,24 +958,45 @@ Types are serialized according to this table:
+|--------------------|-------------------|----------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **`boolean`**      | **`boolean`**     | `true`                           |                                                                                                                                                                                                                                                                                                                           |
+| **`int`**          | **`json int`**    | `1`                              |                                                                                                                                                                                                                                                                                                                           |
+| **`long`**         | **`json long`**   | `1`                              |                                                                                                                                                                                                                                                                                                                           |
+| **`float`**        | **`json float`**  | `1.1`                            |                                                                                                                                                                                                                                                                                                                           |
+| **`double`**       | **`json double`** | `1.1`                            |                                                                                                                                                                                                                                                                                                                           |
+| **`decimal(P,S)`** | **`string`**      | `"0x3162"`                       | Stores the unscaled value, as the two's-complement big-endian binary using the minimum number of bytes, converted to a hexadecimal string prefixed by `0x`                                                                                                                                                                |
```

Comment:
I still think this could be fine as just a decimal string, seems like it would be clearer for a metadata.json reader.

Reply:
I'm fine with either hex string or decimal string.

Reply:
Agreed. The decimal string would be better and easier to work with. We should specify that it _should_ have the correct number of digits after the decimal, even if that requires padding with 0s, like `34.00` for `decimal(9,2)`.